Applications/Spire/Source/TimeAndSales/TimeAndSalesTableView.cpp

```diff
@@ -140,5 +178,45 @@ TableView* Spire::make_time_and_sales_table_view(
         set(TextAlign(properties[i].m_alignment));
     });
   }
+  auto pull_indicator = make_pull_indicator();
+  pull_indicator->setSizePolicy(QSizePolicy::Expanding, QSizePolicy::Fixed);
```

Comment:
Keep construction logic together, in general keep code together instead of spreading it out. The ```make_pull_indicator``` should handle as much of the construction as possible, instead of having some of it constructed in there and then some of it constructed here.